website/pages/en/publishing/publishing-a-subgraph.mdx

```diff
@@ -2,7 +2,7 @@
 title: Publishing a Subgraph to the Decentralized Network
 ---
 
-Once your subgraph has been [deployed to Subgraph Studio](/deploying/deploying-a-subgraph-to-studio), you have tested it out, and are ready to put it into production, you can then publish it to the decentralized network.
+Once your subgraph has been [deployed to Subgraph Studio](/deploying/deploying-a-subgraph-to-studio), you have tested it out, and it is ready to put it into production, you can then publish it to the decentralized network.
```

Comment:
Hmm, I feel like it should either be "you are ready to put it into production" or "it is ready _to be put_ into production". This change reads weird to me.

Reply:
I like "you are ready to put it into production" (this was my own oversight and I missed the "be" that should've been included).

Reply:
```suggestion
Once your subgraph has been [deployed to Subgraph Studio](/deploying/deploying-a-subgraph-to-studio), you have tested it out, and you are ready to put it into production, you can then publish it to the decentralized network.
```